pkg/util/overridemanager/overridemanager.go

```diff
@@ -352,6 +405,50 @@ func applyArgsOverriders(rawObj *unstructured.Unstructured, argsOverriders []pol
 	return nil
 }
 
+func applyFieldOverriders(rawObj *unstructured.Unstructured, FieldOverriders []policyv1alpha1.FieldOverrider) error {
+	if len(FieldOverriders) == 0 {
+		return nil
+	}
+	for index := range FieldOverriders {
+		pointer, err := jsonpointer.New(FieldOverriders[index].FieldPath)
```

Comment:
We can pre-validate the `FieldPath` field in `pkg/webhook/overridepolicy/validating`.

Reply:
Added validating part and its unit test. Also succeeded to run in local host.

Reply:
Solved. Verification both for `FieldPath` and `SubPath`.